packages/contracts-bedrock/deploy/002-L1CrossDomainMessenger.ts

```diff
@@ -37,9 +37,7 @@ const deployFn: DeployFunction = async (hre) => {
 
   const upgradeTx = await Proxy.upgradeToAndCall(
     messenger.address,
-    L1CrossDomainMessenger.interface.encodeFunctionData('initialize(address)', [
-      portal.address,
-    ])
+    L1CrossDomainMessenger.interface.encodeFunctionData('initialize')
```

Comment:
Does this need to be `initialize()` or does it not matter?

Reply:
Doesn't matter